### DISCUSSION TITLE
Error CSS style withing installation process

### DISCUSSION THREAD

**Discussion by neerro** [Kovah/LinkAce#478] [Support]
Hi there!
Within my installation process [Setup without Docker](https://www.linkace.org/docs/v1/setup/setup-without-docker/) on Step.5, I met the next issue (it seems occurs due to CSS style).
Is anybody can help me to fix this issue?

My server configuration:
OS Ubuntu 22.04
Apache v2.4.53
PHP v7.4.29
MariaDB v10.6.7
Instance directory rules:
   chown -R www-data:www-data
   chmod -R 775

![img_linkace](https://user-images.githubusercontent.com/75830569/170838452-fcfd0445-313e-47d8-ac73-4c068bc3232a.jpg)

**Comment by Kovah** [comment]
If you see this you did not properly configure your web server. It must point its docroot to the `public` directory in the LinkAce folder, not the LinkAce folder itself.

**Comment by neerro** [comment]
@Kovah, thanks a lot for your prompt reply.
Yes, definitely, the issue was in it. I've changed docroot path to the `public` and everything worked well.

But during of app usage, unfortunately I've met another one issue (see the picture below and my reproducing steps).
Step 1. From dashboard panel, I have gone to `settings`, made some changes.
Step 2. Went to the end of page to `Save Settings`
Step 3. After I put on the button `Save Settings` the window with message has appeared on address `mydomane.name/fetch/update-check`
Finally, all settings are safe, but the app doesn't back to the settings window.
Is this also something wrong with my settings wherefore I see this bug?
Please, support me with this issue too.

![01](https://user-images.githubusercontent.com/75830569/170968959-f766566f-58f7-4809-b60b-17c4fcf46313.jpg)

**Comment by martadinata666** [comment]
it works correctly in my system, this is the result on step three

![2022-06-15-204728_1666x1047_scrot](https://user-images.githubusercontent.com/2433562/173843563-3fa7df83-c4c2-4219-b644-6b06e4a1895f.png)

My apache conf
```
<VirtualHost *:443>
    Protocols h2 http/1.1
    ServerAdmin webmaster@localhost
    DocumentRoot /var/www/html/public

    ErrorLog ${APACHE_LOG_DIR}/error.log
    CustomLog ${APACHE_LOG_DIR}/access.log combined

    SSLEngine on
    SSLCertificateFile /etc/ssl/certs/ssl-cert-snakeoil.pem
    SSLCertificateKeyFile /etc/ssl/private/ssl-cert-snakeoil.key
    
    <Directory "/var/www/html/">
        AllowOverride All
        Require all granted
    </Directory>

    <FilesMatch "\.php$">
        SetHandler "proxy:fcgi://localhost:9000/"
        SSLOptions +StdEnvVars
    </FilesMatch>
     
    <Files ~ "^.ht">
        Order allow,deny
        Deny from all
    </Files>

</VirtualHost>

```

**Reply by neerro** [reply]
> it works correctly in my system, this is the result on step three
>
Huh! Maybe I did something wrong in my settings? As I haven't any extraordinary in my apache configuration, everything as usually.
And in principle I would to know, what does it mean this error? What kind of issue it could be?
